api/python/provisioner/commands/confstore_export.py

```diff
@@ -98,12 +117,23 @@ def run(self, **kwargs):
 
             Conf.load('provisioner', pillar[PillarKey(pillar_key)])
 
-            for data in template_data:
+            delimiter = '=>'
+            for i,data in enumerate(template_data):
                 if data:
-                    Conf.set("provisioner", data.split(':')[0], data.split(':')[1])
+                    key = data.split(delimiter)[0]
+                    value = data.split(delimiter)[1]
```

Comment:
You can simplify code here to avoid multiple `split`-calls like this
```python
key, value = data.split(delimeter)
```

Reply:
Yes. will update